docs/walkthroughs/provision-keycloak-apb.adoc

```diff
@@ -0,0 +1,96 @@
+
+== Provision
+Choose the type of service you would like to provision:
+
+* Keycloak service
+* Shared Keycloak service
+
+Go to the OpenShift web console. Open the *Mobile* Tab > *Services* from the service catalog and select the Keycloak service.
+This should open the Keycloak service provision wizard. Press the *Next* button and provide the configuration for the KeyCloak service
+
+=== KeyCloak service
+This service type can be shared between shared between different projects
+
+*The configuration will ask for the following parameters:*
+
+* *Add to Project(required)*: Create a new project or point to an existing project
+* *Keycloak admin username(required)*: This user should be able to administer the realm or create new realms
+* *Keycloak admin password(required)*: The password of the admin user for the Keycloak web console
+* *Connect to an existing shared service*: Remain unchecked
+* *URL of the shared service*: Remain empty
+* *Name of the Keycloak realm (defaults to current namespace)*: Remain empty
+
+After the service has been provisioned, you are then able to open the Keycloak web console
+with the address 'http://keycloak-<project-name>.<routing-suffix>' and the ```<project-name>``` realm should be available.
+
+=== Shared KeyCloak service
+This service type is linked to an already existing original KeyCloak instance
+
+*The configuration of a shared service needs the following parameters:*
+
+* *Add to Project(required)*: Create a new project or point to an existing project
+* *Keycloak admin username(required)*: This user should be able to administer the realm or create new realms
+* *Keycloak admin password(required)*: The password of the admin user for the Keycloak web console
+* *Connect to an existing shared service*: Should be checked
+* *URL of the shared service*: This is the url of the existing Keycloak instance we want to link this service to
+* *Name of the Keycloak realm (defaults to current namespace)*: Specify a new realm for the Shared Keycloak service
+
+After the service is provisioned, you are then able to open its web console with the address of the shared service that was specified.
+The realm specified in the configuration should also be available within this Keycloak instance.
+
+[[bind-keycloak-service]]
+== Bind
+Bindings can be created for a provisioned Keycloak service, whether it's a shared one or not.
+
+* Go to the project overview where your Keycloak service is provisioned.
+* Select the Keycloak service under the *Provisioned Services* section. This should open up the service's details.
+* Click *Create Binding* under the *Bindings* section. This should bring up the wizard for bind creation.
+* Fill in the parameters:
+- *Service Name (Required)*: This is the name of the service the client is for. This value will be used for creating the `client
+id` and `name` in the Keycloak realm.
+- *Name of Keycloak realm*: This is the name of the Keycloak realm where you want the client
```

Comment:
Based on our conversation on Friiday, I believe we are removing the realm from the bind option